lib/srv6.c

```diff
@@ -114,3 +125,53 @@ const char *seg6local_context2str(char *str, size_t size,
 		return str;
 	}
 }
+
+struct srv6_locator *srv6_locator_alloc(const char *name)
+{
+	int namelen = 0;
+	struct srv6_locator *locator = NULL;
+
+	locator = XCALLOC(MTYPE_SRV6_LOCATOR, sizeof(struct srv6_locator));
+	if (locator) {
+		namelen = sizeof(locator->name);
+		if (namelen > SRV6_LOCNAME_SIZE - 1)
+			namelen = SRV6_LOCNAME_SIZE - 1;
```

Comment:
These 3 lines are unnecessary, from the strlcpy man page:
```
     strlcpy() copies up to dstsize - 1	characters from	the string src to dst,
     NUL-terminating the result	if dstsize is not 0.
```
Please delete all this and use `strlcpy(locator->name, name, sizeof(locator->name))`

Reply:
I fixed it